### PR TITLE
#165305222 Users should be able to reset password.

### PR DESCRIPTION
**Pull request by katherine95**
#### What does this PR do?
- Implements reset password functionality.

#### Description of Task to be completed?
- Develop user interface for the user interface to allow a user to reset the password.
- Redirect URL sent to user's email to reset password confirmation user interface.

#### How should this be manually tested?
- Clone the repository:
    $ git clone https://github.com/andela/ah-the-jedi-frontend.git

- change directory to the ah-the-jedi-frontend directory:
    $ cd ah-the-jedi-frontend/

- Get all branches:
    $ git pull

- Switch to the feature branch:
    $ ft-user-reset-password-165305222

- Install the project dependencies
    $  npm install

- Run the project
    $ npm run start:dev

- Navigate to reset password page
    $<your-localhost>/reset_password

#### What are the relevant pivotal tracker stories?
[#165305222](https://www.pivotaltracker.com/story/show/165305222)
#### Any background context you want to add?
None
#### Successful password reset
Screenshot 2019-06-05 at 14 57 02

#### Successful password confirm
Screenshot 2019-06-05 at 14 57 39

#### Passwords don't match
Screenshot 2019-06-05 at 14 57 49

#### Already reset password
Screenshot 2019-06-05 at 14 57 59

### CODE REVIEW

**Review by jkamz — CHANGES_REQUESTED** [review]
Hi. Please squash your commits. Thanks.

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (Benkimeric) [inline .vscode/launch.json]: Could you kindly elaborate what is the purpose of this file?

**Review by Benkimeric — COMMENTED** [review]
Nice implementation on resetting a user password. However, would you kindly centre the elements like in this image? I think it will look nicer. 
image

File: .vscode/launch.json (hunk 1 above)
Comment (AlvinMugambi) [reply .vscode/launch.json]: It was automatically updated with the text editor, vscode which was unnecessary. Unnecessary files are now removed.

**Comment by AlvinMugambi** [comment]
> Nice implementation on resetting a user password. However, would you kindly centre the elements like in this image? I think it will look nicer.
> image

Updated

**Pull request merged**